source

```diff
@@ -102671,10 +102678,12 @@ dictionary <dfn>StorageEventInit</dfn> : <span>EventInit</span> {
    <dd>Switch to the <span>data state</span>. Emit the current tag token.</dd>
 
    <dt>EOF</dt>
-   <dd><span>Parse error</span>. Emit an end-of-file token.</dd>
+   <dd><span data-x="parse-error-eof-in-attribute-value">eof-in-attribute-value</span><span>parse
+   error</span>. Emit an end-of-file token.</dd>
 
    <dt>Anything else</dt>
-   <dd><span>Parse error</span>. <span>Reconsume</span> in the <span>before attribute name
+   <dd>This is an <span data-x="parse-error-unexpected-character-in-attribute-value">unexpected-character-in-attribute-value</span>
```

Comment:
Here we are after quoted attribute value and we met new attribute name, e.g.:
```html
<div id="someId"class="123">
```
So error should be `missing-whitespace-after-attribute-value`